e2e/commands/start/start_test.go

```diff
@@ -44,7 +41,6 @@ var _ = Describe("running binary Horusec with start parameter", func() {
 		session, err = gexec.Start(cmd, GinkgoWriter, GinkgoWriter)
 		Expect(err).NotTo(HaveOccurred())
 		session.Wait(testutil.AverageTimeoutAnalyzeForExamplesFolder)
-		Expect(session).Should(gexec.Exit(0))
```

Comment:
This check shouldn't be added to those tests that was already running?

Reply:
I have added a new check in the line below.

Reply:
my bad